README.md

```diff
@@ -0,0 +1,70 @@
+🤗 Evaluate is a library that aims to make evaluating and comparing models and reporting their performance easier and more standardized. 
+
+It currently contains:
+
+- **implementations of dozens of popular metrics**: the existing metrics cover a variety of tasks spanning from NLP to Computer Vision, and include dataset-specific metrics for datasets such as [GLUE](https://huggingface.co/datasets/glue) and [SQuAD](https://huggingface.co/datasets/squad). With a simple command like `bleu_score = load_metric("bleu")`, get any of these metrics ready to use for evaluating a ML model in any framework (Numpy/Pandas/PyTorch/TensorFlow/JAX).
+- **an easy way of pushing to the 🤗 Hub**: you can create new metrics and push them to a dedicated Space in the 🤗 Hub with `evaluate-cli create [metric name]`, which allows you to see easily compare different metrics and their outputs for the same sets of references and predictions. 
+
+[🎓 **Documentation**](https://huggingface.co/docs/metrics/)
+
+[🔎 **Find a metric in the Hub**](https://huggingface.co/metrics) [🌟 **Add a new dataset to the Hub**](https://github.com/huggingface/rvaluate/blob/master/ADD_NEW_DATASET.md)
+
+🤗 Evaluate also has lots of useful features like:
+
+- Type checking to make sure that you are using the right input formats, depending on the metric structure and arguments.
+- Metric cards that describe the values, limitations and ranges of metrics, providing examples of their usage and usefulness.
+- [*Something else here*]
+
+# Installation
+
+## With pip
+
+🤗 Evaluate can be installed from PyPi and has to be installed in a virtual environment (venv or conda for instance)
+
+```bash
+pip install evaluate
+```
+
+## With conda
+
+🤗 Datasets can be installed using conda as follows:
+
+```bash
+conda install -c huggingface -c conda-forge evaluate
+```
+
+For more details on installation, check the installation page in the documentation: https://huggingface.co/docs/evaluate/installation
+
+## Installation to use with PyTorch/TensorFlow/pandas
+
+If you plan to use 🤗 Evaluate with PyTorch (1.0+), TensorFlow (2.2+) or pandas, you should also install PyTorch, TensorFlow or pandas.
+
+For more details on using the library with NumPy, pandas, PyTorch or TensorFlow, check the quick start page in the documentation: https://huggingface.co/docs/evaluate/quickstart
+
+# Usage
+
+🤗 Evaluate's main methods are:
+
+- `evaluate.list_metrics()` to list the available metrics
+- `evaluate.load_metric(metric_name, **kwargs)` to instantiate a metric
+

```

Comment:
I think we can also add an example e.g. with accuracy where we use `compute`.

Reply:
done!